classes/02class/exercises/c02-network08/drii-cavalcanti/c02-network08.txt

```diff
@@ -0,0 +1,15 @@
+1. What is the difference between inbound x outbound?
+   The difference between inboud(ingress) and outbound(egress) is the direction of the traffic based on the point-of-view of the subnets interface. In other words, inbound is the traffic that an instance receives and outbound is the traffic that an instance sends.
+    
+2. What is going to happen by default? Everything allowed or denied?
+   By default, each custom network ACL denies all inbound and outbound traffic until you add rules.
+
+3. How would you block SSH connection comming from the network 172.16.200.200/24?
+   Inbound
+   Rule#   Type	             Protocol	Port Range	 Source	                Allow/Deny
+   100     SSH               TCP        22           172.16.200.200/24	    DENY
+
+4. How would you block access from your network to an external network 180.200.50.0/24?
+   Outbound   
+   Rule#   Type	             Protocol	Port Range	 Destination	        Allow/Deny
+   100     All IPv4 traffic  All        All          180.200.50.0/24	    DENY
```

Comment:
There is a minor detail here to be fixed. Similar to the above.

Remember that ACL rules are ordered. So, should you use rule # 100?